config/spring19_seminar/cola_var/cola_bert_ccg.conf

```diff
@@ -0,0 +1,77 @@
+// Import the defaults using the relative path
+include "../defaults.conf"
+
+
+// Output path
+project_dir = ${JIANT_PROJECT_PREFIX}
+exp_name = exp_pretrain
+run_name = run_bert_ccg
+
+
+// Logistics
+cuda = 0
+random_seed = 42
+
+
+// Rebuilding
+allow_reuse_of_pretraining_parameters = 0 // If this is 1 OR do_target_task_training == 0 then 
+                                          // `task_names_to_avoid_loading` will be empty; see `main.py` line 271, line 345
+reload_tasks = 0     // If true, force the rebuilding of the task files in the experiment directory
+reload_indexing = 0  // If true, force the rebuilding of the index files in preproc
+reload_vocab = 0     // If true, force the rebuilding of the vocabulary files in the experiment directory
+reindex_tasks = ""
+
+
+// Optimization
+lr = 3e-4  // this performs reasonablely well in practice
+lr_patience = 4  // number of epochs between last validation improvement and lr annealing, following final.conf
+patience = 20  // number of epochs between last validation improvement and early stopping, following final.conf
+max_vals = 10000 // following final.conf
```

Comment:
If you're following final.conf in a lot of places, just inherit from there instead of from defaults.conf.

Reply:
Also, probably best to use batch_size=16 everywhere. It's the default in jiant, and it's what @sheng-fu and @HaokunLiu are using.

Reply:
@sleepinyourhat  now it's a batch_size of 16. Also used including final.conf